### PR TITLE
orders plugin graphql docs

### PR DESCRIPTION
**Pull request by MohanNarayana**
Signed-off-by: Mohan Narayana <mohan.narayana@mailchimp.com>

### CODE REVIEW

File: docs/orders.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Order
+
+Orders plugin deals with order creation, fetching orders by order ID, reference ID or payment ID, cancelling order, refunds.
```
Comment (delagroove) [inline docs/orders.md]: 'and refunds.' sound best

Comment (delagroove) [reply docs/orders.md]: @MohanNarayana

Comment (MohanNarayana) [reply docs/orders.md]: Done

File: docs/orders.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  
+- status: String! - The machine-readable order status.
+  
+- summary: OrderSummary! - A summary of the totals for all fulfillment groups for this order
+  
+- totalItemQuantity: Int! - Total quantity of all items in the order
+  
+- updatedAt: DateTime! - The date and time at which this order was last updated
+  
+- surcharges: [AppliedSurcharge]!
```
Comment (delagroove) [inline docs/orders.md]: add def

File: docs/orders.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## Query
+1. **orders** : Query the Orders collection for one or more shops. Accepts a list of shopIds as query parameter. An order has much of the same information that its source cart had, but additionally tracks the process of fulfilling the order and completing the associated payments, as well as returns, taxes, and invoice information.
+
+2. **orderById**  
+Query the Orders collection for an order. It requires both orderId (as id) and shopId as query parameters. A token string can be used as a parameter for annonymous orders not linked with an account.
```
Comment (delagroove) [inline docs/orders.md]: typo

File: docs/orders.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+
+3. **orderByReferenceId**   
+Query the Orders collection for an order. It requires both referenceId (as id) and shopId as query parameters. A token string can be used as a parameter for annonymous orders not linked with an account. ReferenceId id the one displayed to the customer on the storefront or in the order confirmation email after successfully placing an order.
```
Comment (delagroove) [inline docs/orders.md]: typo

Comment (delagroove) [reply docs/orders.md]: id vs is

File: docs/orders.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+5. **refunds** : Query the Orders collection for an order, and returns refunds applied to payments associated with this order. Requires both orderId and shopId. [More on refunds](https://github.com/reactioncommerce/docs/blob/3860ad31862c515dc9a9bcf8d9f86d781ad85f17/docs/fulfilling-orders.md#refunds).
+
+6. **refundsByPaymentId** : An order can have one or more payment method. This query looks up the Orders collection for an order, and returns refunds applied to a specific payment associated with this order. [More on payments](https://github.com/reactioncommerce/docs/blob/3860ad31862c515dc9a9bcf8d9f86d781ad85f17/docs/fulfilling-orders.md#payments).
+
+## Mutation
+1. **addOrderFulfillmentGroup** : Use this mutation to add a new order fulfillment group to an order. It must have at least one item, which can be provided or moved from another existing group. The built-in fulfillment types are Shipping, Pickup, and Digital. Each fulfillment type has at least one fulfillment method. For example, “shipping” type can have methods like Free Shipping and Flat Rate USPS. [More on fulfilment](https://github.com/reactioncommerce/docs/blob/trunk/docs/fulfilling-orders.md#fulfillment)
```
Comment (delagroove) [inline docs/orders.md]: typo fulfillment

File: docs/orders.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+2. **cancelOrderItem** : Use this mutation to cancel one item of an order, either for the full ordered quantity or for a partial quantity. If partial, the item will be split into two items and the original item will have a lower quantity and will be canceled. If this results in all items in a fulfillment group being canceled, the group will also be canceled. If this results in all fulfillment groups being canceled, the full order will also be canceled.
+
+3. **createRefund** : Use this mutation to create a refund on an order payment. A refund maybe partial or full and needs to be supported by the payment method. A successful refund is reflected in the order page. 
+
+4. **moveOrderItems** : This mutation is used to change the fulfillment group of an order. Each fulfillment group has atlease one fulfillment type and the fulfillment type describes the method of delivery of the item.
```
Comment (delagroove) [inline docs/orders.md]: typo at least

File: docs/orders.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+5. **placeOrder** : Used it place the final order. The input for this mutation `PlaceOrderInput` takes order and payments information. The order will be placed only if authorization is successful for all submitted payments.
```
Comment (delagroove) [inline docs/orders.md]: typo

File: docs/orders.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+6. **sendOrderEmail** : A mutation that compiles and server-side renders the email template with order data, and sends the email. Email is triggered upon creation of order, when shipped, creation of order refund refund and creation of item refund.
```
Comment (delagroove) [inline docs/orders.md]: remove one refund

File: docs/orders.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+7. **splitOrderItem** : Use this mutation to reduce the quantity of one item of an order and create a new item for the remaining quantity in the same fulfillment group, and with the same item status. You may want to do this if you are only able to partially fulfill the item order right now. It required ItemId and quantity of the item to be split.
```
Comment (delagroove) [inline docs/orders.md]: typo

**Review by delagroove — CHANGES_REQUESTED** [review]

**Comment by brent-hoover** [comment]
I think everything here should just be in the GraphQL schema. One of the big pluses of GraphQL is that it is self-documenting so I think having this copied to another place just creates a maintenance issue. I would suggest we just double-check that all this information is in the schema itself and then close it.

**Comment by delagroove** [comment]
Actually i have checked the docs and we brought that info from there. closing.

**Pull request closed without merging**